local-tests/tests/wrapped-keys/testFailEthereumSignTransactionWrappedKeyWithInvalidParam.ts

```diff
@@ -70,11 +70,7 @@ export const testFailEthereumSignTransactionWrappedKeyWithInvalidParam = async (
         id,
       });
     } catch (e: any) {
-      if (
-        e.message.includes(
-          'Error executing the Signing Lit Action: Error: When signing transaction- invalid hexlify value'
-        )
-      ) {
+      if (e.message.includes('invalid hexlify value')) {
```

Comment:
Made this less specific since the exact string isn't important and this is now being properly prefixed consistently due to refactors in error handling in the LA in this PR